### ISSUE TITLE
[Screen Reader-Home Page-Navigation]Name Property is not defined for the Video Play button for videos showing under Get inspired section.

### ISSUE THREAD

**Issue by v-swasha** [Microsoft/pxt#2595]
**User Experience:**
If any control have incorrect role defined then the person with visual related disability will find it difficult to interpret the proper fields information. This will hinder him to access all the controls of the page efficiently and also not compatible with AT.

**Test Environment**
OS: RS2 Version 1703(OS Build 15063.483)
Platform: Edge.

**Prerequisites:** Turn on Narrator

**Repro Steps**
1. Navigate to https://makecode.com/
2. Navigate through all controls (say computing education, about Microsoft makecode, get inspired, resources elements etc)lying on the page
3. Navigate to Video showing under Get Inspired section.
4-Verify, If Name property is defined for the video play button or not.

**Actual Result**
Name Property is not defined for the Video Paly button for videos showing under Get inspired section.

**Expected Result**
Name Property should be defined for the Video Paly button for videos showing under Get inspired section.

**MAS Reference**
https://microsoft.sharepoint.com/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={248054a6-5e68-4771-9e1e-242fb5025730}

**Suggested fix**
Using ARIA role to expose the control type of user interface component
e.g.
< div role="toolbar" tabindex="0" id="customToolbar" >
< img src="img/btn1.gif" role="button" tabindex="-1" alt="Home" id="b1" title="Home" >
Please Refer attachment for more details
![mas4 1 2_role is defined incorrect for project screen](https://user-images.githubusercontent.com/30459830/28661921-fe768f88-72d5-11e7-8f13-039982cb8522.png)

**Please refer the attachment:**
![mas4 1 2_homepage_navigation](https://user-images.githubusercontent.com/30459830/28813474-047bb2de-76b7-11e7-8cc9-855bd256f4c6.jpg)
[MAS4.1.2_HomePage_Navigation.zip](https://github.com/Microsoft/pxt/files/1189759/MAS4.1.2_HomePage_Navigation.zip)

**Comment by veler** [comment]
This is part of the YouTube embed feature and we cannot make any change to it.

**Comment by veler** [comment]
Needs to be forwarded to YouTube.

**Comment by abchatra** [comment]
External youtube bug
